commands/publish/__tests__/publish-command.test.js

```diff
@@ -259,11 +256,10 @@ Map {
   describe("from-package", () => {
     it("publishes unpublished packages", async () => {
       const testDir = await initFixture("normal");
-      const project = new Project(testDir);
 
-      getUnpublishedPackages.mockImplementationOnce(async () => {
-        const pkgs = await project.getPackages();
-        return pkgs.slice(1, 3);
+      getUnpublishedPackages.mockImplementationOnce(packageGraph => {
+        const pkgs = packageGraph.rawPackageList.slice(1, 3);
+        return pkgs.map(pkg => packageGraph.get(pkg.name));
```

Comment:
another mostly-unrelated change due to refactoring `getUnpublishedPackages()`. it definitely exposes that my mock isn't abstract enough, and should probably be a network mock instead...